solr/core/src/java/org/apache/solr/packagemanager/PackageManager.java

```diff
@@ -148,7 +167,7 @@ private boolean deployPackage(SolrPackageInstance packageInstance, boolean pegTo
 
       // Get package params
       try {
-        boolean packageParamsExist = ((Map)PackageUtils.getJson(solrClient.getHttpClient(), solrBaseUrl + "/api/collections/abc/config/params/packages", Map.class)
+        boolean packageParamsExist = ((Map)PackageUtils.getJson(solrClient.getHttpClient(), solrBaseUrl + "/api/collections/"+collection+"/config/params/packages", Map.class)
```

Comment:
Static path names such as `/api/collections` and also `/config/params/packages` shold be well defined constants, not loose strings. I see them repeated many places.

Reply:
Fixed, thanks.